tests/test_opd.py

```diff
@@ -209,7 +216,85 @@ def test_opd_wavelength():
             galsim.config.SetupExtraOutput(config)
 
 
+def test_opd_phase():
+    """Test that we can add a phase that zeros out the OPD"""
+    with TemporaryDirectory() as d:
+        # Write out an OPD at the bandpass effective wavelength
+        config = textwrap.dedent(
+            f"""
+            input:
+                telescope:
+                    file_name: LSST_r.yaml
+            image:
+                bandpass:
+                    file_name: LSST_r.dat
+                    wave_type: nm
+            output:
+                dir: {d}
+                opd:
+                    file_name: opd1.fits
+                    fields:
+                        - {{thx: 1.121 deg, thy: 1.231 deg}}
+                    eps: 0.612
+            """
+        )
+        config = yaml.safe_load(config)
+        galsim.config.ProcessInput(config)
+        galsim.config.SetupExtraOutput(config)
+        galsim.config.SetupConfigFileNum(config, 0, 0, 0)
+        galsim.config.extra.WriteExtraOutputs(config, None)
+        fn = Path(d) / "opd1.fits"
+        hdr = fits.getheader(fn)
+
+        zk = np.zeros(29)
+        for i in range(1, 29):
+            zk[i] = hdr[f'AZ_{i:03d}']
+
+        # Write a new config with phases to zero out the OPD
+        config = textwrap.dedent(
+            f"""
+            input:
+                telescope:
+                    file_name: LSST_r.yaml
+                    fea:
+                        extra_zk:
+                            zk: {(zk*1e-9).tolist()}
+                            eps: 0.612
```

Comment:
Is there a test we can add that checks that this changes the in-focus image too?  I think it should, and as discussed on the telecon, I would have thought that could be used to add a slight random perturbation to the aberrations similar to what we used to do with Bo's model.

Reply:
Sure, I can add a test that just checks that the PSF isn't identical before and after adding a screen.

The difference between adding the phase here and what's in [OptWF](https://github.com/LSSTDESC/imSim/blob/main/imsim/atmPSF.py#L37) is that the current PR is changing the wavefront uniformly over the full field of view, and the optical perturbations from Bo's model are field angle dependent.  

One could certainly extend the current PR to from adding a constant (over-the-field) single Zernike phase to adding a DoubleZernike that captures the field dependence.  But I think that's more than I need at the moment (and really begs for GalSim 2.5 too).

Reply:
I added a test that the RubinOptics photon op gives different output after adding the constant phase.  I think that's sufficient and easier than going all the way to an image.